tf2onnx/onnx_opset/nn.py

```diff
@@ -649,7 +652,8 @@ def version_1(cls, ctx, node, **kwargs):
             raise ValueError("input channel must be positive")
         k_output_channels = k_input_channels * k_channel_multiplier
 
-        node.set_attr("kernel_shape", [k_h, k_w])
+        if k_h >= 0 and k_w >= 0:
```

Comment:
Is zero a valid value for k_h or k_w? Would it be better to change to 'k_h > 0 and k_w > 0'?

Reply:
Good point, I'm not really sure. Zero-sized dimensions are valid in general, but I'm not sure it makes sense for convolution kernels. Interestingly, I just tried it in my TensorFlow environment (2.4.1) and a zero-sized convolution kernel makes hang (probably in an infinite loop) with CPU and log and error and crash with GPU.

Reply:
So should we change >= to > here?

Reply:
Yes, I have made both suggested changes.